package.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "docs",
     "version": "1.0.0",
-    "description": "Corpweb built on HUGO",
-    "private": true,
+    "description": "Datadog Docs built on HUGO",
+    "private": false,
```

Comment:
Might want to keep this as true to avoid accidently publishing to the npm public repo. When its private you need to have a user login